src/pages/FlightsList/FlightsList.js

```diff
@@ -1,5 +1,108 @@
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import FlightFilter from './components/FlightFilter';
+import FlightInfoCard from '../../components/FlightInfoCard/FlightInfoCard';
+
 const FlightsList = () => {
-  return <h1>FlightsList</h1>;
+  const [flightInfos, setFlightInfos] = useState();
+  const [filterOptions, setFilterOptions] = useState({});
+
+  const toggleFilterOption = (filterKey, toggleName) => {
+    setFilterOptions(prevFilterOptions => ({
+      ...prevFilterOptions,
+      [filterKey]: [...prevFilterOptions[filterKey]].map(({ name, checked }) =>
```

Comment:
```suggestion
      [filterKey]: prevFilterOptions[filterKey].map(({ name, checked }) =>
```
어차피 map은 새로운 배열을 리턴하기 때문에 이 경우 스프레드를 사용할 필요가 없겠습니다

Reply:
아! 그렇네요 리뷰 반영해 수정하겠습니다!! 감사합니다.!